executor/Makefile

```diff
@@ -51,6 +51,17 @@ compile_tensorflow_proto:
 	cd proto/tensorflow/core && go mod init github.com/tensorflow/tensorflow/tensorflow/go/core && cd -
 	go build ./proto/tensorflow/serving
 
+
+compile_kfserving_proto:
+	git clone https://github.com/NVIDIA/triton-inference-server
```

Comment:
This is out of the scope of this PR, but at the moment Triton is using its own package (`nvidia.inferenceserver`) for the KFServing protos. It would be good if every V2 Dataplane server used a unified one (defined in the KFServing repo), otherwise it will be hard to mix & match different V2 servers.

That's something to change on the Triton side though. And I imagine it may not be trivial, since they also have other Triton-specific messages defined in [`grpc_service.proto`](https://github.com/NVIDIA/triton-inference-server/blob/master/src/core/grpc_service.proto).

Reply:
This has been updated now